subt/main.py

```diff
@@ -419,7 +419,13 @@ def return_home(self, timeout, home_threshold=None):
                     desired_direction += math.pi  # symmetry
                     for angle in self.joint_angle_rad:
                         desired_direction -= angle
-                self.bus.publish('desired_altitude', target_z)
+
+                d = distance3D(self.xyz, [target_x, target_y, target_z])
+                time_to_target = d/self.max_speed
+                desired_z_speed = (target_z - self.xyz[2]) / time_to_target
+                #print(desired_z_speed)
+                #self.bus.publish('desired_altitude', target_z)
```

Comment:
I chat I meant these debug print/obsolete code

Reply:
done in  e703c67